Orange/regression/tree.py

```diff
@@ -101,8 +101,11 @@ def _score_cont():
             if score == 0:
                 return  REJECT_ATTRIBUTE
             score *= non_nans / len(col_x)
-            branches = (col_x > cut).astype(int)
+
+            with np.errstate(invalid="ignore"):  # NaN in col_x handled later
```

Comment:
Wouldn't it be cleaner to initialize branches to -np.ones and set only the non-nan values after that.
Maybe it's just me, but I prefer not triggering errors to declaring which ones to ignore.